Dockerfile

```diff
@@ -1,28 +1,27 @@
-FROM ubuntu:xenial
+FROM ubuntu:bionic
 
-LABEL authors='Chris Rust <chris.rust@solarwinds.com>, Dawid Śmiech <dawid.smiech@solarwinds.com>'
-
-ARG swisnap_version
-ARG swisnap_repo=swisnap
+LABEL authors='SolarWinds AppOptics team <support@appoptics.com>'
 
 USER root
 ENV DEBIAN_FRONTEND noninteractive
 
-RUN \
-  apt-get update && \
-  apt-get -y install software-properties-common && \
-  LC_ALL=C.UTF-8 add-apt-repository -y ppa:rmescandon/yq && \
-  apt-get update && \
-  apt-get -y upgrade && \
-  apt-get -y install \
-    apt-transport-https \
-    ca-certificates \
-    docker.io \
-    curl \
-    yq
+ARG swisnap_repo=swisnap
+RUN apt-get update && \
+    apt-get -y install software-properties-common && \
+    LC_ALL=C.UTF-8 add-apt-repository -y ppa:rmescandon/yq && \
+    apt-get update && \
+    apt-get -y upgrade && \
+    apt-get -y install \
+      apt-transport-https \
+      ca-certificates \
+      docker.io \
+      curl \
+      yq && \ 
+      apt-get clean && \
+      rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
-RUN \
-  echo "deb https://packagecloud.io/solarwinds/${swisnap_repo}/ubuntu/ xenial main" > /etc/apt/sources.list.d/swisnap.list && \
+ARG swisnap_version
+RUN echo "deb https://packagecloud.io/solarwinds/${swisnap_repo}/ubuntu/ xenial main" > /etc/apt/sources.list.d/swisnap.list && \
```

Comment:
> xenial

 vs. 

> FROM ubuntu:bionic

Reply:
@d-maslyk   I'm not sure what the state of this PR is and what we are wanting.

I see that the snap version I upgraded to isn't available in bionic.  Since the new snap agent is available, I did a test to use focal image and focal snap agent `agent_3.0.1.685` but it doesn't respect the writting to  stdout via `exec ${SWISNAP_HOME}/sbin/swisnapd --config ${CONFIG_FILE} ${FLAGS} --log-path ""`. 

Pod logs show
```
time="2020-05-26T14:17:50Z" level=info msg="Will log to default file: /var/log/SolarWinds/Snap/swisnapd.log"

Reply:
yes, this is expected for now. Please do not update agent to version 3.0.x in this PR, it will be in next and requires more work. Updating to latest 2.7.5 is ok. Packagecloud has bionic version for it.